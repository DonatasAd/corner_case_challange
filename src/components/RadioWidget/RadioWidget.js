import React from 'react';
import styles from './RadioWidget.module.css';
import Navbar from './Navbar/Navbar';
import Tabbar from './Tabbar/Tabbar';
import RadioStations from './RadioStations/RadioStations';

export default function RadioWidget() {
  return (
    <div className={styles.RadioWidget}>
      <Navbar />
      <RadioStations />
      <Tabbar />
    </div>
  );
}
