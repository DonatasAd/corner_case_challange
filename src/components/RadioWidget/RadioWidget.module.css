.RadioWidget {
  display: flex;
  flex-direction: column;
  overflow: hidden;
  background-color: #2a2a35;
  height: 35rem;
  width: 23rem;
  margin: 2rem;
  border-radius: 2rem;
  box-shadow: 0px 35px 0px -26px #14141b, 0px 40px 0px -22px #16161d, 0px 40px 0px -16px #191921, 0 40px 0 -10px #1c1c24, 0 40px 0 0px #20202a;
}
