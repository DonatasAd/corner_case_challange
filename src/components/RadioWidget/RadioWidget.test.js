import React from 'react';
import { shallow } from 'enzyme';
import RadioWidget from './RadioWidget';

describe('RadioWidget', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<RadioWidget />);
    expect(component).toMatchSnapshot();
  });
});
