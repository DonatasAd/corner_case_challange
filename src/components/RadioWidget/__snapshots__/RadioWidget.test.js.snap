// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioWidget should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
